sdk/communication/AzureCommunicationCalling/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 2.0.1-beta.1 (2021-09-09)
+
+### Bugs Fixed
+- When trying to accept a terminated call `accept` API call will fail with appropriate error.
```

Comment:
Recommend:
- `CallingClient.accept` will now throw errors when trying to accept a terminated call or if already in an active call.